docs/content/TopNQuery.md

```diff
@@ -211,4 +211,7 @@ Users who can tolerate *approximate rank* topN over a dimension with greater tha
     "queryType": "topN",
     "threshold": 2
 }
-```
\ No newline at end of file
+```
+
+### DimExtractionFn
```

Comment:
Couldn't this also be done as a TopN over the actual "ids" and then a remapping in the Broker on the way out?